OVERVIEW.md

```diff
@@ -0,0 +1,123 @@
+# Overview
+
+The `hydra-torch` repo serves three purposes.
+
+1. It demonstrates a standardization for creating and organizing Hydra configuration classes for the ML/torch community and beyond.
+	- If someone wants to create their own `hydra-<library>` repo, `hydra-torch` points towards an upper bound on what it should look like.
+2. It unifies a collection of classes across projects in one place, ensuring robust testing, version compatibility, and PyPI distributed packaging for each.
+    - We see many hydra users reimplementing these classes (and not tracking APIs of the configured projects). `hydra-torch` factors this code out.
+3. It provides best practices and guidance on how to organize code and utilize Hydra for research, production, and other innovative use cases.
+
+##### Terminology for this overview:
+- The overall effort and repo is named:`hydra-torch`
+- **Library:** The library being configured
+  - e.g. `torch`, `torchvision`
+- **Project:** The project corresponding to the library being configured
+  - e.g. `hydra-configs-torch`, `hydra-configs-torchvision`
+- **Package:** The installable package containing the configs
+  - here, also `hydra-configs-torch`, `hydra-configs-torchvision`. find definition in the project dir's `setup.py`.
+
+
+
+### Repo Structure
+```
+📂 hydra-torch
+└ 📁 configen 				# source files used when generating package content. all projects have their own subdirectory here.
+└ 📁 hydra-configs-torch		# a project corresponding to configuring a specific library (contains package definition)
+└ 📁 hydra-configs-torchvision		# "
+└ 📁 hydra-configs-<future-library> 	# "
+└ 📁 examples				# use-cases and tutorials
+```
+
+Each `hydra-configs-<library-name>` defines its own package corresponding to a project it provides classes for. That is, `hydra-configs-torch` contains a package (with its own `setup.py`) which provides classes for `torch`.
+
+[`hydra-configs-projects.txt`](hydra-configs-projects.txt) specifies a list of projects this repo maintains. When adding a project, please update this file.
```

Comment:
I think we can drop this, adding a project is not going to happen frequently and whoever is doing it will discover it easily enough.